spec/exporters/formatters/utaac_decision_indexable_formatter_spec.rb

```diff
@@ -29,29 +28,11 @@
   include_context "schema with humanized_facet_value available"
 
   it_behaves_like "a specialist document indexable formatter"
-  it_behaves_like "a tribunal decision indexable formatter"
 
   it "should have a type of utaac_decision" do
     expect(formatter.type).to eq("utaac_decision")
   end
 
-  context "without hidden_indexable_content" do
-    it "should have body as its indexable_content" do
-      allow(document).to receive(:body).and_return("body text")
-
-      allow(document).to receive(:hidden_indexable_content).and_return(nil)
-      expect(formatter.indexable_attributes[:indexable_content]).to eq("body text")
-    end
-  end
-
-  context "with hidden_indexable_content" do
-    it "should have hidden_indexable_content as its indexable_content" do
-      allow(document).to receive(:body).and_return("body text")
-      allow(document).to receive(:hidden_indexable_content).and_return("hidden indexable content text")
-
-      indexable = formatter.indexable_attributes[:indexable_content]
-      expect(indexable).to eq("hidden indexable content text\nbody text")
-    end
-  end
+  include_examples "tribunal decision hidden_indexable_content"
```

Comment:
Added shared examples testing behaviour of `hidden_indexable_content`. Added same shared examples to other tribunal decision indexable formatter specs.